little_chassis/Motion motion
little_chassis/IoState io
little_chassis/Battery battery
uint32 error_code
